src/Map/src/Bridge/Google/tests/GoogleOptionsTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Bridge\Google\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\UX\Map\Bridge\Google\GoogleOptions;
+use Symfony\UX\Map\Bridge\Google\Option as GoogleOption;
+
+class GoogleOptionsTest extends TestCase
+{
+    public function testWithMinimalConfiguration(): void
+    {
+        $options = new GoogleOptions();
+
+        self::assertSame([
```

Comment:
More "$this" in Symfony codebase no ?

Reply:
IDK, `assert*` are static methods so I want to call them with `self::` :D 

But that's a long debate, I will switch to `$this->` if necessary (or if Fabbot says so :eyes:)

Reply:
I prefer too ;)  but code style is code style i guess ^^

Reply:
I'm seing both syntax in Symfony UX repository, IMO let's just don't care about this until we configure https://cs.symfony.com/doc/rules/php_unit/php_unit_test_case_static_method_calls.html

Reply:
Anyway, even in Symfony they use both syntax:
<img width="1297" alt="image" src="https://github.com/user-attachments/assets/a52c1b95-0f6d-493f-a41d-6b56ac5989fd">
<img width="1301" alt="image" src="https://github.com/user-attachments/assets/91f67425-22c0-443f-b10d-18af968e0945">